android/fastlane/Fastfile

```diff
@@ -23,59 +23,98 @@ platform :android do
 
   JS_PACKAGE = load_json(json_path: "../package.json")
 
-  desc "Build a Staging APK and AAB"
-  lane :staging do
+  desc "GPS: Build a Staging AAB"
+  lane :staging_gps do
     gradle(
-      task: "clean assembleStagingDebug",
+      task: "clean assemble",
+      build_type: "gpsStaging",
       print_command: false,
     )
   end 
 
-  desc "Build a Release APK and AAB"
-  lane :release do
+  desc "GPS: Build a Release AAB"
+  lane :release_gps do
 
-    increment_version_code(
-      gradle_file_path: "../android/app/build.gradle",
-    )
+    build_number =  ENV["GITHUB_RUN_NUMBER"]
```

Comment:
The idea here is to set the version code automatically from the workflow run number. `VersionName` updates are infrequent and can be done manually in `build.gradle`